options/validations.go

```diff
@@ -0,0 +1,37 @@
+package options
+
+func clusterOptValidate(key string, value interface{}) bool {
```

Comment:
This function name is a little misleading as you are only validating the value here. What if `clusterDefaultOptions[key]` doesn't exist ? Won't this return true ?